### DISCUSSION TITLE
just discussion

### DISCUSSION THREAD

**Discussion by JorenVsyntra** [community/community#145289] [New to GitHub]
Hey everyone!

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @JorenVsyntra Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
